nuxeo-drive-client/nxdrive/wui/conflicts.py

```diff
@@ -7,17 +7,102 @@
 from nxdrive.wui.dialog import WebDialog, WebDriveApi
 from nxdrive.wui.translator import Translator
 from PyQt4 import QtCore
+from Queue import PriorityQueue
+from time import sleep
+from datetime import timedelta, datetime
+from nxdrive.engine.workers import Worker
+
 
 log = get_logger(__name__)
 
 
+class UserNameResolver(Worker):
```

Comment:
I think this is a bit too complicated. If I understand it correctly, it refreshes the user's (full) name at 1d interval after the last refresh, but the refresh could be staggered, since different users's names are needed at different times. 
1. if a user names is used several times a day, it may only be retrieved 1st time
2. if a user name is used every few days, it will be retrieved every time is used
I think the code would be simpler, if we simply refresh once a day, e.g. using one of those Poll workers; this may be slightly more often than in case 2. but it's only 1 request/day.